src/uwtools/drivers/orog_gsl.py

```diff
@@ -19,6 +20,19 @@ class OrogGSL(DriverTimeInvariant):
 
     # Workflow tasks
 
+    @task
+    def input_config_file(self):
+        """
+        The input config file.
+        """
+        path = self._input_config_path
+        yield self.taskname(str(path))
+        yield asset(path, path.is_file)
+        yield None
+        inputs = [str(self.config["config"][k]) for k in ("tile", "resolution", "halo")]
+        with writable(path) as f:
+            print("\n".join(inputs), file=f)
```

Comment:
```suggestion
        with open(path, "w", encoding="utf-8") as f:
            print("\n".join(inputs), file=f)
```
I know `writable()` is convenient, but using it implies that `path` might be `None`, in which case it will write to `stdout`, which I doubt is something we'd ever want to happen in this context. To avoid any chance of bizarre bugs, I think a few more characters to make our yes-really-a-disk-file intent explicit are worth it.

Reply:
I was really enjoying a tiny moment of lazy here! 😉